tests/integration/helpers/element-test.js

```diff
@@ -239,7 +239,7 @@ module('Integration | Helper | element', function (hooks) {
     assert.dom('p#content').hasText('Test').hasClass('extra');
   });
 
-  test('it can be invoked inline', async function (assert) {
+  test.skip('it can be invoked inline', async function (assert) {
```

Comment:
As discussed in team meeting today with @ef4 and @rwjblue, this failing test reveals an issue in Ember or Glimmer VM itself and will be tracked separately.